### PR TITLE
add levels of DFT automation script

### PR DESCRIPTION
**Pull request by lancekavalsky**
This PR adds a script for plotting the levels of DFT automation analysis

### CODE REVIEW

File: scripts/plot_levels_of_automation.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        "font.size": 18,
+        "axes.linewidth": 2.0,
+        "lines.dashed_pattern": (5, 2.5),
+        "legend.edgecolor": "black",
+        "lines.markersize": 8,
+        "lines.linewidth": 2.0,
+    }
+)
+
+AUTOMATED = np.array([2, 1, 1, 1, 1, 9, 1])
```
Comment (hegdevinayi) [inline scripts/plot_levels_of_automation.py:52]: Please add a description/explanation of this variable. Perhaps even with a simpler example/reference to the paper.

File: scripts/plot_levels_of_automation.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+TRADITIONAL = 60 * np.array([16, 9, 10, 9, 9, 3, 2])
```
Comment (hegdevinayi) [inline scripts/plot_levels_of_automation.py:53]: Please add a description/explanation of this variable. Perhaps even with a simpler example/reference to the paper.

File: README.md [hunk 3]
```diff
@@ -32,11 +32,15 @@ The repository is organized as follows:
...
 
-        If run as-is, the script reproduces the 3-paneled FIG.~5 in the main text.
+        If run as-is, the script reproduces the 3-paneled FIG. 5 in the main text.
 
     * `plot_acceleration_from_sequential_learning__ALL_ACQ.py`: Similar to the previous script; plots and compares statistics from all acquisition functions considered (MLI, MU, random, space-filling) for all SL tasks.
 
-        If run as-is, the script reproduces the 3-paneled FIG.~S1 in the Supplementary Information.
+        If run as-is, the script reproduces the 3-paneled FIG. S1 in the Supplementary Information.
+
+    * `plot_levels_of_automation.py`: Script for plotting the cumulative time  of executing the DFT pipeline at varying levels of automation.
```
Comment (hegdevinayi) [inline README.md:41]: Not for this PR, but please add the above line to the `supplementary.tex` file in the paper as well.

**Review by hegdevinayi — CHANGES_REQUESTED** [review]
Looks good to me. Requesting a couple of comments (/variable docstrings) in the plotting script.

File: README.md (hunk 3 above)
Comment (lancekavalsky) [reply README.md:41]: thanks -- good catch. done

**Review by hegdevinayi — APPROVED** [review]
The explanation is great, thanks! This PR LGTM

**Pull request merged**